state-client-lib/src/config.rs

```diff
@@ -15,12 +15,17 @@ pub struct SCEnvCLIConfig {
     /// Default confirmations
     #[arg(long, env)]
     pub sc_default_confirmations: Option<usize>,
+
+    /// Maximum size of a decoded message
+    #[arg(long, env, default_value_t = 4 * 1024 * 1024)]
```

Comment:
@GCdePaula I was thinking that it might be better to setup the default to 100Mb, so we can avoid configuring these for the time being.

Reply:
I broke the node with an input of 500Kb; with Guilherme's estimation, it is just ~$100. https://github.com/cartesi/rollups-contracts/issues/77

Reply:
Done!